### PR TITLE
Adds docs for UP Chime support

### PR DESCRIPTION
**Pull request by AngellusMortis**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/71874
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifiprotect.markdown [hunk 1]
```diff
@@ -151,6 +151,13 @@ Each UniFi Protect door lock will get a device in Home Assistant with the follow
 * **Device Configuration** - Door locks will get configuration controls for the Auto-Lock Timeout, selecting the Paired Camera, and Status Light switch
 * **Button** - A disabled by default button is added for each door lock device. The button will let you reboot your door lock device.
 
+### UniFi Protect Smart Chime
+
+Each UniFi Protect smart chime will get a device in Home Assistant with the following:
+
+* **Button** - A button to trigger the chime manually for each smart chime device. Also a disabled by default button is added that will let you reboot your smart chime device.
+* **Device Configuration** - Smart chimes will get a volume slider adjust the loudness of the chime as well as sensor the last time the chime rang.
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Device Configuration** - Smart chimes will get a volume slider to adjust the loudness of the chime as well as a sensor for the last time the chime rang.
```

File: source/_integrations/unifiprotect.markdown [hunk 2]
```diff
@@ -197,6 +204,15 @@ Use to dynamically set the message on a Doorbell LCD screen. This service should
...
 | `duration`             | Yes      | Number of minutes to display the message for before returning to the default message. The default is to not expire. |
 
+### Service unifiprotect.set_chime_paired_doorbells
+
+Use to set the paired doorbell(s) with a smart chime.
+
+| Service data attribute | Optional | Description                                                                                                  |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------ |
+| `entity_id`            | No       | The smart chime you want to pair doorbells with (actually the "Play Chime" button).                          |
+| `doorbells`            | No       | A [target selector](https://www.home-assistant.io/docs/blueprint/selectors/#target-selector) for any number of doorbells you want to pair to the chime. |
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown]: Please don't use full URLs; additionally: I'm also not sure how the target selector is relevant to mention or link in this case?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/unifiprotect.markdown [hunk 3]
```diff
@@ -151,6 +151,13 @@ Each UniFi Protect door lock will get a device in Home Assistant with the follow
 * **Device Configuration** - Door locks will get configuration controls for the Auto-Lock Timeout, selecting the Paired Camera, and Status Light switch
 * **Button** - A disabled by default button is added for each door lock device. The button will let you reboot your door lock device.
 
+### UniFi Protect Smart Chime
+
+Each UniFi Protect smart chime will get a device in Home Assistant with the following:
+
+* **Button** - A button to trigger the chime manually for each smart chime device. Also a disabled by default button is added that will let you reboot your smart chime device.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Button** - A button to trigger the chime manually for each smart chime device. Also, a disabled by default button is added to let you reboot your smart chime device.
```

File: source/_integrations/unifiprotect.markdown [hunk 4]
```diff
@@ -151,6 +151,13 @@ Each UniFi Protect door lock will get a device in Home Assistant with the follow
...
+* **Device Configuration** - Smart chimes will get a volume slider to adjust the loudness of the chime as well as a sensor for the last time the chime rang.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Device Configuration** - Smart chimes will get a volume slider to adjust the chime's loudness and a sensor for the last time the chime rang.
```

**Review by bdraco — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**